packages/documentation/src/stories/components/alert/web-component/post-alert.stories.ts

```diff
@@ -0,0 +1,142 @@
+import { Meta, StoryContext, StoryFn, StoryObj } from '@storybook/web-components';
+import { html } from 'lit';
+import { BADGE } from '../../../../../.storybook/constants';
+import { spreadArgs } from '../../../../utils';
+import './post-alert.styles.scss';
+
+const meta: Meta<HTMLPostAlertElement> = {
+  title: 'Components/Post Alert',
+  component: 'post-alert',
+  render: renderAlert,
+  decorators: [externalControl],
+  parameters: {
+    badges: [BADGE.NEEDS_REVISION],
+  },
+  args: {
+    dismissible: false,
+    dismissLabel: 'Dismiss',
+    fixed: false,
+    innerHTML: '<p>Contentus momentus vero siteos et accusam iretea et justo.</p>',
+  },
+  argTypes: {
+    dismissLabel: {
+      if: {
+        arg: 'dismissible',
+      },
+    },
```

Comment:
```suggestion
    },
          type: {
        name: 'string',
        required: true,
      },
```
Add the required. Maybe it could be added from stencil?

Reply:
I added this to the meta, not the web component since it is not always mandatory.